libyul/optimiser/Suite.h

```diff
@@ -32,6 +32,13 @@
 #include <string_view>
 #include <memory>
 
+#undef OUTPUT_PERFORMANCE_METRICS
+//#define OUTPUT_PERFORMANCE_METRICS 1
```

Comment:
Maybe it would be better to add a CMake option for this?

Reply:
I mean we could also use something like an environment variable.
We could always record the time taken per step (I don't think that's a performance drawback).
And if a certain environment variable is set (which we could also re-use for other things) AND the total compilation time is above 2 seconds or so, output the table.

Reply:
I think a compile-time switch may be better - after all, we can we even safely print to stdout in emscripten?